build.gradle

```diff
@@ -111,7 +111,8 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 ext {
-    reformLogging= '5.0.1'
+    reformLogging= '5.1.0'
```

Comment:
bump to latest see (hmcts/ccd-definition-store-api#458)

Reply:
done